src/pages/overview/AllOverview.tsx

```diff
@@ -7,12 +7,14 @@ import { OverviewGroup } from './group'
 import { IndicatorsCardGroup } from './Indicators'
 import { OrganisationUnitsCardGroup } from './OrganisationUnits'
 import { OtherCardGroup } from './Other'
+import { ProgramsAndTrackersCardGroup } from './ProgramsAndTracker'
 import { ValidationsCardGroup } from './Validations'
 
 export const AllOverview = () => {
     return (
         <OverviewGroup title={i18n.t('Metadata management')}>
             <CategoryCardGroup showTitle={true} />
+            <ProgramsAndTrackersCardGroup showTitle={true} />
```

Comment:
Maybe put this after validations so that it matches the order that's in the sidebar?